src/main/java/uk/gov/hmcts/darts/authentication/model/Permission.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.darts.authentication.model;
```

Comment:
I think the model package (3 classes) here should be `authorisation` instead of authentication?

Reply:
Good spot